apps/ws/src/app.module.ts

```diff
@@ -26,6 +26,4 @@ if (process.env.SENTRY_DSN) {
   controllers: [AppController],
   providers,
 })
-export class AppModule implements OnModuleInit {
-  async onModuleInit() {}
-}
+export class AppModule {}
```

Comment:
Hey @scopsy I do not familiar with this module is this base implementation is unnecessary?

Reply:
@djabarovgeorge it's just a hook nestjs provides, you can read more about it [here](https://docs.nestjs.com/fundamentals/lifecycle-events). We used it previously but not used here anymore.